esp/templates/program/modules/resourcemodule/classrooms.html

```diff
@@ -74,3 +97,71 @@ <h2>Step 3: Classrooms</h2>
 {% endifequal %}
 
 <br />
+
+<!--This branch has a fix for when the form is originally hidden-->
+<script src="https://rawgit.com/elo80ka/django-dynamic-formset/f5f68c818953ddf43ffa7cb2e9aa391c3d4dfb47/src/jquery.formset.js"></script>
+<!--jQuery.initialize plugin is created to help maintain dynamically created elements on the page-->
+<script src="https://rawgit.com/pie6k/jquery.initialize/master/jquery.initialize.js"></script>
+<script>
+<!--Formset settings-->
```

Comment:
This is JS, so you want `//` or `/*` comments -- I think these technically work too since the JS is directly in the HTML, but JS-style comments are much more readable/expected and will continue to work in a standalone script.

Reply:
Doh, that's what I get for going back and forth between JS and HTML.